doc/source/whatsnew/v2.1.0.rst

```diff
@@ -304,6 +304,7 @@ Deprecations
 - Deprecated literal string/bytes input to :func:`read_html`. Wrap literal string/bytes input in ``io.StringIO`` / ``io.BytesIO`` instead. (:issue:`53767`)
 - Deprecated option "mode.use_inf_as_na", convert inf entries to ``NaN`` before instead (:issue:`51684`)
 - Deprecated parameter ``obj`` in :meth:`GroupBy.get_group` (:issue:`53545`)
+- Deprecated parsing datetimes with mixed time zones unless user pass ``utc=True`` to :func:`to_datetime`, in a future version this will raise a warning and will advise to use ``utc=True`` (:issue:`50887`)
```

Comment:
you're already raising a warning - perhaps you meant that in the future it will raise an error? I think you don't need that part of the sentence anyway (from "in a future" until the issue number)

Reply:
thanks, I removed the part of the sentence from "in a future" until the issue number.